.github/workflows/build.yml

```diff
@@ -103,9 +103,13 @@ jobs:
       - name: Install Rust
         if: env.GIT_DIFF
         uses: dtolnay/rust-toolchain@stable
-      - name: Move the cl-vault
+
+      # This is the new one
+      - name: Move smart contracts
         if: env.GIT_DIFF
-        run: mkdir -p smart-contracts/contracts/cl-vault/test-tube-build/wasm32-unknown-unknown/release && mv ./cl_vault.wasm  smart-contracts/contracts/cl-vault/test-tube-build/wasm32-unknown-unknown/release/cl_vault.wasm
+        run: |
+          find . -name "*.wasm" -exec bash -c 'file="{}"; contract_name=$(basename "$file" .wasm); mkdir -p "smart-contracts/contracts/$contract_name/test-tube-build/wasm32-unknown-unknown/release"; mv "$file" "smart-contracts/contracts/$contract_name/test-tube-build/wasm32-unknown-unknown/release/$contract_name.wasm"' \;
```

Comment:
This will fail as the name of the contract for example is "cl_vault" but our naming convention at the contract level is "cl-vault"